### ISSUE TITLE
Quasar dev -m ssr reload inject html content in js and css files

### ISSUE THREAD

**Issue by mirkofisic** [quasarframework/quasar#12870]
### What happened?

I use SSR mode in which I set ssr middleware, every change inside ssr middleware quasar watch changes and try to reload server and build/compile new files, but every **.js** and **.css** file is **injected with index.html content**, this is happend only if something is changed in **ssr-src** folder, if I change something in src folder everything is reloaded without problem.

So that means that **render middleware** is currently processing requests for all static files such as **.js** and **.css** files and filling them with **index.html content**.

[Screens1](https://user-images.githubusercontent.com/6698925/159128678-0575382c-9047-4ef7-802a-240583409380.png)
[Screens2](https://user-images.githubusercontent.com/6698925/159128718-ac45f23b-c796-49b9-ad2e-bd841e5ef09c.png)
[Screens3](https://user-images.githubusercontent.com/6698925/159166814-cb996fee-e203-47b5-9c13-1b61eded85e4.png)

**I can't reproduce this problem on codepan because it doesn't allow SSR.**

### What did you expect to happen?

css and js files do not need to be processed within the render middleware.

### Reproduction URL

https://codesandbox.io/s/github/quasarframework/quasar-codesandbox/tree/master

### How to reproduce?

1. quasar add mode ssr
2. create your own ssr middleware
3. add middleware file in quasar config 
4. execute **quasar dev -m ssr**
5. change somting in your middleware file and wait to reload

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

SSR Mode

### Platforms/Browsers

Chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19043) - win32/x64
NodeJs - 16.14.0

Global packages
  NPM - 8.4.1
  yarn - 1.22.4
  @quasar/cli - 1.3.1
  @quasar/icongenie - Not installed
  cordova - 8.1.2 (cordova-lib@8.1.1)

Important local packages
  quasar - 2.6.1 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-webpack - 3.4.3 -- Quasar Framework App CLI with Webpack
  @quasar/extras - 1.13.3 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.14
  pinia - 2.0.12 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.17.8 -- Babel compiler core.
  webpack - 5.70.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.7.4 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - LAPTOP-GDNUB9MA
  Ethernet 12 - 192.168.56.1
  Wi-Fi - 192.168.0.14
```

**Comment by mirkofisic** [comment]
As I see this seems to be resolved in version 2.6.6, thank you very much :)

**Comment by yusufkandemir** [comment]
It is fixed in [@quasar/app-webpack-v3.5.0](https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fapp-webpack-v3.5.0)
